core/src/main/java/org/elasticsearch/client/Client.java

```diff
@@ -420,36 +414,6 @@
     MultiTermVectorsRequestBuilder prepareMultiTermVectors();
 
     /**
-     * Percolates a request returning the matches documents.
-     */
-    ActionFuture<PercolateResponse> percolate(PercolateRequest request);
```

Comment:
that is true. The docs that described how to integrate with percolate api via the Java client have been replaced with: https://www.elastic.co/guide/en/elasticsearch/client/java-api/master/java-specialized-queries.html#java-query-percolate-query

I'll bring back the percolator java api docs with a description on how to access the api now that the percolator api has been removed from the `Client` interface. Otherwise people that wish to use the percolator api a bit longer are in the dark before cutting over to the search api.